### ISSUE TITLE
[투빅스]-스터디 분야

### ISSUE THREAD

**Issue by JunnGT** [D2CAMPUS-PARTNER/8th-PARTNER-s-DAY#33]
#### 참석자 정보 
 - 이준걸 / 대학원 1기 / i201331749@gmail.com
- 강인구 / 4학년 / kangig94@gmail.com

#### 동아리 소개 
 - 저희 동아리는 먼저 10주 동안 신입기수를 위한 머신러닝/딥러닝 집중교육이 있습니다. 이후에는 머신러닝/딥러닝/텍스트마이닝/강화학습 등 세부 세미나를 선택하여 심화교육을 이어갑니다. 또한 학기말에 있을 컨퍼런스를 위해 팀을 꾸려 프로젝트를 진행합니다. 한 정식 교육과정 외에 관심이 있고, 알아가고 싶은 부분이 있다면 스터디를 꾸리거나 참여해 공부할 수 있습니다.
투빅스 홈페이지 : http://www.datamarket.kr/
투빅스 소개 : https://drive.google.com/open?id=1ZND0dwRBTEF0HB8Uthh5wrK06tSr_3_U
투빅스 프로젝트 : http://www.datamarket.kr/xe/board_pdzw77

#### 2018년 하반기 동아리 활동 및 수상 내역  
 -
빅콘테스트 챔피언 리그 한국정보화진흥원장 상
 2회 미래에셋대우 빅데이터페스티벌 장려상
 제7회 문화관광 빅데이터 분석 대회 은상
정보보호 R&D 데이터 챌린지 3등
제1회 우정사업본부 데이터사이언스 빅데이터 해커톤 우수상

#### 2019 상반기 활동계획
- 신입생 세미나
 -0주차 : R과 파이썬 예습하기
1주차 : 회귀분석 / 로지스틱 회귀 / Git 
2주차 : 나이브베이즈 / SVM / KNN
3주차 : 의사결정나무 / 앙상블 / 클러스터링
4주차 : 차원축소  LDA / PCA
5주차 : Neural Network / 프레임워크
6주차 : 크롤링 / 텍스트 기초
7주차 : 텍스트 심화
8주차 : 이미지 기초
9주차 : 이미지 심화
10주차 : 강화학습
11주차 : 프로젝트 주제 발표

- 텍스트 세미나
- Lecture 3 | GloVe: Global Vectors for Word Representation
- Lecture 4: Word Window Classification and Neural Networks
- Lecture 5: Backpropagation and Project Advice
- Lecture 9: Machine Translation and Advanced Recurrent LSTMs and GRUs
- Lecture 10: Neural Machine Translation and Models with Attention
- Lecture 12: End-to-End Models for Speech Processing
- Lecture 16: Dynamic Neural Networks for Question Answering

-강화학습 세미나
1.intro
2. MDP
3. Planning by Dynamic Programming
4. Model Free Prediction
5. Model Free Control
6. Value Function Approximation
7. Policy Gradient Methods
8. Integrating Learning and Planning
9. Exploration and Exploitation
10. Case Study : Classic Games

-머신러닝 세미나
- 회귀분석 심화
- GAM
- 분석 프레임워크, NN's cost function  
- 앙상블
- San Francisco Crime Classification - 1 (Kaggle)
- Google Analytics Customer Revenue Prediction (Kaggle)
- function optimization and support vector machine
- 경제경영 데이터 분석(Dacon 3)
- Recommender System
- Structuring Machine Learning Projects 
- principle of classification(Bayes classifier&bayes error

#### 기타 
 -
